elixir-mode.el

```diff
@@ -185,7 +185,9 @@ for the Elixir programming language."
     `(
       (atoms . ,(rx ":"
                     (or
-                     (one-or-more (any "a-z" "A-Z" "_" "\"" "'"))
+                     (and
+                      (any "a-z" "A-Z" "_" "\"" "'")
+                      (zero-or-more (any "a-z" "A-Z" "0-9" "_" "\"" "'")))
```

Comment:
The old regexp was:

```lisp
(or 
  (one-or-more (any "a-z" "A-Z" "_" "\"" "'"))

...)
```

Which matches only `:msisdn` in the `:msisdn2pid` atom.

By changing it to 

```lisp
(or 
  (and
    (any "a-z" "A-Z" "_" "\"" "'")
    (zero-or-more (any "a-z" "A-Z" "0-9" "_" "\"" "'")))
```

I ask for the first character in `'("a-z" "A-Z" "_" "\"" "'")` and the subsequent ones in `'("a-z" "A-Z" "0-9" "_" "\"" "'")`. This in turn matches the whole `:msisdn2pid` atom.

Sorry, I should've added more context to the PR.